app/src/main/java/vn/sun/music_68/ui/splashscreen/SplashActivity.java

```diff
@@ -0,0 +1,19 @@
+package vn.sun.music_68.ui.splashscreen;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.os.Handler;
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+
+import vn.sun.music_68.ui.main.MainActivity;
+
+public class SplashActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        startActivity(new Intent(SplashActivity.this,MainActivity.class));
```

Comment:
Nếu anh muốn startActivity có thể làm theo cách này: https://medium.com/capital-one-tech/a-better-way-to-launch-activities-on-android-8a1045181b16
Nếu khó hiểu, đơn giản, sử dụng theo conventions của công ty: https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md#211-argument-in-activity-and-fragment